cluster-autoscaler/loop/trigger.go

```diff
@@ -118,6 +142,20 @@ func StartPodObserver(ctx context.Context, kubeClient kube_client.Interface) *Un
 	}
 }
 
+// triggerNextIteration logs a message if the next iteration was not triggered by unschedulable pods appearing, else it logs a message that the next iteration was triggered by unschedulable pods appearing
```

Comment:
I think we can change it to `logTriggerReason`. I do not have preferences about either.
Ref: [https://github.com/kubernetes/autoscaler/pull/7271#discussion_r1772877092](https://github.com/kubernetes/autoscaler/pull/7271#discussion_r1772877092)

To my understanding, we only need this select statement because we would like to log "Autoscaler loop triggered by unschedulable pod appearing" over any other reason if this is true, because this was the existing behavior before the changes that this PR introduces. Thus, if we do not look at the `unschedulablePodChan`, the next loop will be triggered immediately anyhow, but we would log the reason of "lower priority" for why the next loop was triggered.

Reply:
I don't see a lot of value in prioritizing logging one reason over another like this, but if you want to keep it, that's ok with me. Thanks for renaming.